packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -73,6 +73,7 @@ export type ViewConfig = $ReadOnly<{
       phasedRegistrationNames: $ReadOnly<{
         captured: string,
         bubbled: string,
+        skipBubble?: ?boolean,
```

Comment:
@lunaleaps seems like this property name is (accidentally?) different here, flow check during React sync caught this

Reply:
Oh my mistake! How should I address? Can we fix forward on the sync?

Reply:
Submitting a fix now